File: guanabara-java/src/com/danillo/algoritmos/lista05/Exercicio57.java
package com.danillo.algoritmos.lista05;

import java.util.Scanner;

public class Exercicio57 {
    public static void main(String[] args) {
        Scanner entrada = new Scanner(System.in);

        float somaSalarioHomens = 0, somaSalarioMulheres = 0;
        char continuar = 'S';

        while(continuar == 'S'){
            System.out.println("Insira o salário do funcionário: ");
            float salario = entrada.nextFloat();
            System.out.println("Insira o gênero do funcionaŕio: ");
            char genero = entrada.next().charAt(0);

            if(genero == 'M'){
                somaSalarioHomens = somaSalarioHomens + salario;
            } else {
                somaSalarioMulheres = somaSalarioMulheres + salario;
            }

            System.out.println("Deseja continuar? (S ou N)");
            continuar = entrada.next().charAt(0);
        }

        System.out.println("Total de salários pagos aos homens: R$ " +somaSalarioHomens+
                           "\nTotal de salários pagos às mulheres: R$ " +somaSalarioMulheres);
    }
}

File: guanabara-python/src/lista05/exercicio57.py
continuar = 1
i = 0
soma_salario_homens = 0
soma_salario_mulheres = 0

while continuar == 1:
    salario = float(input("Insira o salário: "))
    genero = int(input("Insira o gênero: 1 - Masculino 2 - Feminino"))

    if genero == 1:
        soma_salario_homens = soma_salario_homens + salario
    else:
        soma_salario_mulheres = soma_salario_mulheres + salario

    continuar = int(input("Deseja continuar? 1 - Sim | 2 - Não"))

print(f"Total salário homens: {soma_salario_homens}.\nTotal salário mulheres: {soma_salario_mulheres}.")
